app/controllers/form_responses_controller.rb

```diff
@@ -18,29 +18,29 @@ def create
       render_422
     end
 
-    render json: @form_response, serializer: FormResponseSerializer, user: current_user
```

Comment:
Why is it replaced? This controller should work with FormResponseSerializer

Reply:
We have general logic on step when we update element it must return orderable element. We need discuss with Martin to split separate controllers depends where action performed. To also handle activities and permissions.